__tests__/app.test.js

```diff
@@ -23,13 +22,28 @@ describe('Get all topics', () => {
             ]);
         });
     });
+});
 
-    test('checking to make sure it returns the right amount of items', () => {
+describe('/api (getting all endpoints)', () => {
+    test('making sure that its responding with an object of endpoints', () => {
         return request(app)
-        .get('/api/topics')
+        .get('/api')
+        .expect(200)
+        .then((topics) => {
+            expect(Object.keys(topics.body.endPoints)).toEqual([ 'GET /api', 'GET /api/topics', 'GET /api/articles' ]);
+        });
+    });
+
```

Comment:
Awesome job, you will need to remember to update this test with each endpoint that you add.
It would be perfectly acceptable to require in the endpoints.json and assert that your response from the server matches the json.